Scripts/generateDocumentation.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+TARGET=$1
+
+if [[ "$TARGET" = "StreamChat" ]];then
+    TARGET_DIRECTORY="Sources/StreamChat"
+elif [[ "$TARGET" = "StreamChatUI" ]]; then
+    TARGET_DIRECTORY="Sources/StreamChatUI"
+else
+    echo "Please specify target to generate docs for (StreamChat or StreamChatUI)"
+    exit 1
+fi
+
+# Move to project root directory
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$scriptDir/../"
+
+# As of May 2021, swift-doc doesn't support directories, so we find all the subdirectories instead
+find "$TARGET_DIRECTORY" -type d -print > Scripts/StreamChat_folder_directories.txt
```

Comment:
The nitty problem with this approach is we need somehow to figure out the emissions which it creates (iterates through assets folders and Generated folder, which we don't need)

Reply:
Fixed.